app/validators/metadata_presenter/validate_answers.rb

```diff
@@ -22,9 +22,9 @@ def validators
       components.map { |component|
         component_validations(component).map do |key|
           "MetadataPresenter::#{key.classify}Validator".constantize.new(
-            {
-              component: component,
-              page_answers: page_answers
+            **{
```

Comment:
What does the ** do?

Reply:
It specifies passing keyword arguments instead of a hash see https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/